docs/sources/administration/team-management/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+aliases:
+  - 
```

Comment:
The aliases need to be moved from https://github.com/grafana/grafana/pull/86942/files#diff-de47761a6b83e1cd74a87c05843e44b98549e5752c2234829dd174977de94ae2